game/src/info/trip.rs

```diff
@@ -852,10 +850,7 @@ fn make_trip_details(
     if path_impossible {
         col.push("Map edits have disconnected the path taken before".text_widget(ctx));
     }
-    // TODO This just needs too much more work
```

Comment:
You did too much work - yay!

Reply:
The "work" here really meant getting proper elevation data, because the original attempt at SRTM wound up with extremely silly values, breaking the plot. So @eldang did the hard work here. ;)